backend/src/main/java/io/debezium/configserver/model/ConnectConnectorConfigResponse.java

```diff
@@ -0,0 +1,40 @@
+package io.debezium.configserver.model;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ConnectConnectorConfigResponse {
+
+    public ConnectConnectorConfigResponse() {
+    }
+
+    private String name;
+
+    public ConnectConnectorConfigResponse(String name) {
+        this.name = name;
+    }
+
+    public ConnectConnectorConfigResponse(String name, Map<String, ?> config) {
+        this.name = name;
+        setConfig(config);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    private Map<String, String> config;
```

Comment:
Same here, all fields to the top, please.

Reply:
check!